src/mixins/common-mixin.ts

```diff
@@ -49,6 +49,29 @@ export class CommonMixin extends Vue {
     }
   }
 
+  /**
+   * Returns request action text for the the specified code.
+   * See namex -> api/namex/resources/name_requests/report_resource.py::_get_request_action_cd_description()
+   */
+  requestActionCdToText (cd: RequestCode): string {
+    switch (cd) {
+      case RequestCode.NEW: return 'New Business'
+      case RequestCode.MVE: return 'Continuation In'
+      case RequestCode.REH: return 'Restoration or Reinstatement'
+      case RequestCode.AML: return 'Amalgamation'
+      case RequestCode.CHG: return 'Name Change'
+      case RequestCode.CNV: return 'Alteration'
+      // case RequestCode.DBA:
+      // case RequestCode.ASSUMED:
+      // case RequestCode.REN:
+      // case RequestCode.REST:
```

Comment:
It's unclear whether these are used atm.
For now, these cases will fall through and display the code instead of wording, which I think is OK.
I can always PR another change quickly if this needs to change.